server/store/missions.js

```diff
@@ -3,15 +3,11 @@ const { getBid } = require('./bids');
 const { getRequest } = require('./requests');
 const { createMissionUpdate } = require('./mission_updates');
 
-const getMission = async missionId => {
```

Comment:
We still need `getMission`, it is used elsewhere.

How about refactoring the code you added so that we have both `getMission` and `getLatestMission`. The code for getting the actual mission doesn't need to be duplicated in both, so `getLatestMission` can call `getMission`.